src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,6 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_SET = new Prefix("set");
```

Comment:
I'm not sure if "set" is a prefix and need to add into here. Based on my understanding, we only need to add new prefix when we need to create a new field in a person model. Like if we create a field to record remarks, then we need to add "r/" as a prefix. But "set" does not function in this way. When user key in "set ...", then program will execute the "set", not to add the values behind to a field.

Please correct me if I'm wrong, thanks 😃

Reply:
Indeed, it should not be a prefix !